distributed/deploy/ssh.py

```diff
@@ -89,7 +89,19 @@ def __init__(
         self.kwargs = copy.copy(kwargs)
         self.name = name
         self.remote_python = remote_python
-        self.nprocs = self.kwargs.pop("nprocs", 1)
+        if kwargs.get("nprocs") is not None and kwargs.get("n_workers") is not None:
+            raise ValueError(
+                "Both nprocs and n_workers were specified. Use n_workers only."
+            )
+        elif kwargs.get("nprocs") is not None:
+            warnings.warn(
+                "The nprocs argument will be removed in a future release. It has been "
+                "renamed to n_workers.",
+                FutureWarning,
+            )
+            self.n_workers = self.kwargs.pop("nprocs")
```

Comment:
Not sure if this is needed or not

```suggestion
            self.n_workers = self.kwargs.pop("nprocs", 1)
```

Reply:
It shouldn't be needed, but wouldn't hurt to add it.